example/src/App.js

```diff
@@ -2,8 +2,16 @@ import * as React from "react";
 import { Tabs, Tab } from 'react-bootstrap';
 import {withBarong, LoginForm, RegisterForm } from "react-barong";
```

Comment:
add space after {

Also, I suppose you should name Login and Register forms as BarongLoginForm and BarongRegisterForm